docs/en_US/NAS/NasGuide.md

```diff
@@ -117,35 +42,13 @@ trainer.train()  # training
 trainer.export(file="model_dir/final_architecture.json")  # export the final architecture to file
```

Comment:
`reward_accuracy` is not defined.   should be `top1_accuracy`

Reply:
@ultmaster could you answer this question?

Reply:
In the ENAS exmaple, `reward_accuracy` is the same as `top1_accuracy`, although they can be different (e.g., `reward_fn` can be a joint function of accuracy and latency). For this documentation, I'm okay with changing into `top1_accuracy` to avoid possible confusion.

Reply:
sure, will update to `top1_accuracy`